### DISCUSSION TITLE
Course/content created by sub org creator is not visible to reviewer of same org.

### DISCUSSION THREAD

**Discussion by snehalsabade** [Sunbird-Ed/Community#720] [Bugs]
Hello Team,
@gandham-santhosh  @AmiableAnil 

**Sunbird Version - 4.10.0**
I am facing the issue with review the content/course created by same (subOrg). 
I have one scenario -
I have one rootOrg and in that there are two subOrgs. When a course or content is created by a subOrg creator and sent for review, the rootOrg reviewer can access and review it. However, the subOrg reviewer is unable to access or review this course/content.
How it work it is a functionality or bug.
How can I resolve this issue?

cc @paritshivani @komalm @manishakapadnis

**Comment by gandham-santhosh** [comment]
@vinukumar-vs can you please check this?

**Comment by ashokreddy1208** [comment]
@snehalsabade This is expected functionality. 

When a user creates a content it always belongs to root org (inspite of content creator belongs to RootOrg or SubORg).
So only the RootOrg reviewer can only review the contents(because contents belongs to RootOrg only).

cc @vinukumar-vs @amitpriyadarshi @rajeevsathish @AmiableAnil

**Reply by snehalsabade** [reply]
Hello Team,
Thank you so much for your support.
I am grateful for the support you've provided. Thanks to your guidance.

cc @paritshivani @komalm @manishakapadnis
